code-challenges.test.js

```diff
@@ -26,27 +26,101 @@ const numbersArray2 = [24, 27, 30, 33, 36]
+    // before colon will return if condition is T
+    // after colon will return if condition is F
+    return numObj.number % 3 === 0 ? `${numObj.number} is divisible by three` : `${numObj.number} is not divisible by three`
+}
+
+// Explain your code: These two functions will return a string indicating whether the object passed's number property is divisible by 3 or not.
 
 // --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
 
 // a) Create a test with expect statements for each of the variables provided.
-
-const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
-// Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
-const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
-// Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
+// describe("functionName", () => {
+//     it("short description of what fxn should do", () => {
+//       expect(functionName(input)).toEqual(output)
+//     })
+//   } )
+describe("capitalizer", () => {
+    const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+    // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
+    const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
+    // Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
+
+    it("returns an array with all the first letters cap'd", () => {
+        expect(capitalizer(randomNouns1)).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"])
+        expect(capitalizer(randomNouns2)).toEqual(["Temperature", "Database", "Chopsticks", "Mango"])
+    })
+})
+// ReferenceError: capitalizer is not defined
 
 // b) Create the function that makes the test pass.
-
-// Explain your code:
+// input: arr of words
+// output: arr of words with first letter cap'd
+// logic: 
+// .map() to iterate through arr and return arr of exact same length
+// .split() method to turn word to arr of letters
+// inner .map() to iterate through individual letters
+// .toUpperCase() to capitialize letter if the index is 0
+// .join() on inner map
+
+// const capitalizer = (strArr) => {
+//     return strArr.map(word => {
+//         let letterArr = word.split("") // this should give me strArr.length many arrays with each letter being an index
+//         return letterArr.map((letter, i) => {
+//             if (i === 0) {
+//                 return letter.toUpperCase()
+//             } else {
+//                 return letter
+//             }
+//         }).join("")
+//     })
+// }
+
+// going to attempt to make more concise
+const capitalizer = (strArr) => {
+    return strArr.map(word => word.split("").map((letter, i) => i === 0 ? letter.toUpperCase() : letter).join(""))
+}
+
+// Explain your code: Both of these functions will return an array of strings with the first letter capitalized in every string.
```

Comment:
Love that you show your refactoring. I bet you can make it even more concise!  (don't overthink it). 
Great work, Morgan! 🎉